packages/@react-stately/table/src/useTableColumnResizeState.ts

```diff
@@ -87,11 +87,11 @@ export function useTableColumnResizeState<T>(props: TableColumnResizeStateProps<
       columnLayout.recombineColumns(state.collection.columns, uncontrolledWidths, uncontrolledColumns, controlledColumns)
   , [state.collection.columns, uncontrolledWidths, uncontrolledColumns, controlledColumns, columnLayout]);
 
-  let onColumnResizeStart = useCallback((key: Key) => {
+  let startResize = useCallback((key: Key) => {
     setResizingColumn(key);
   }, [setResizingColumn]);
 
-  let onColumnResize = useCallback((key: Key, width: number): Map<Key, ColumnSize> => {
+  let computeResizedColumns = useCallback((key: Key, width: number): Map<Key, ColumnSize> => {
```

Comment:
Maybe `updateResizedColumns` would be slightly better because this does have a side effect of updating the uncontrolled widths in addition to just computing the sizes...